ivanti_security_controls/icon_ivanti_security_controls/actions/start_patch_scan/action.py

```diff
@@ -39,19 +38,25 @@ def run(self, params={}):
             "templateId": params.get(Input.TEMPLATE_ID),
             "useMachineCredential": use_machine_credential
         }
-        scan_details = self.connection.ivanti_api.start_patch_scan(payload)
-
-        i = 0
-        # Poll for patch scan completion
-        while i < max_poll_time:
-            time.sleep(10)
-            patch_scan_status_details = self.connection.ivanti_api.get_patch_scan_status_details(scan_details['id'], allow_404=True)
-            if patch_scan_status_details is not None:
-                if patch_scan_status_details.get('isComplete'):
-                    scan_details['isComplete'] = patch_scan_status_details['isComplete']
-                    scan_details['updatedOn'] = patch_scan_status_details['updatedOn']
-                    break
-            i += 10
+        
+        self.connection.ivanti_api.create_session_credential()
+
+        scan = self.connection.ivanti_api.start_patch_scan(payload)
+
+        operation_location_url = scan.headers.get("Operation-Location")
```

Comment:
Might be worth adding a check for `None` based on some of the inconsistencies and changes over time we have seen with this API.